src/Meditation.Core/Services/AttachableProcessListProvider.cs

```diff
@@ -17,8 +17,7 @@ public AttachableProcessListProvider(IProcessListProvider processListProvider)
                 .Where(id => processListProvider.TryGetProcessById(id, out _))
                 .Select(pid => new ProcessInfo(
                     processListProvider.GetProcessById(pid).ProcessInternal, 
-                    isNetCoreApp: true, 
-                    isNetFramework: false))
+                    ProcessType.NetCoreApp))
```

Comment:
is it temporary (while we not support framework) right?

Reply:
Already fixed in more recent commit